autodoc.pl

```diff
@@ -498,13 +498,28 @@ ($$)
 
     my $file_is_C = $file =~ / \. [ch] $ /x;
 
-    # Count lines easier
-    my $get_next_line = sub { $line_num++; return <$fh> };
+    # Count lines easier and handle apidoc continuation lines
+    my $get_next_line = sub {   my $contents = <$fh>;
+                                return unless defined $contents;
+
+                                $line_num++;
+                                while ($contents =~ s/ ^ ( =for \ apidoc .*)
+                                                           \s* \\ \n 
+                                                     /$1/x)
+                                {
+                                    my $next = <$fh>;
+                                    last unless defined $next;
```

Comment:
This needs a `no warnings 'exiting';`

Reply:
Your comment doesn't make sense to me.  The `last` is exiting a loop.  The code works without warnings